torch/serialization.py

```diff
@@ -453,6 +453,12 @@ def persistent_load(saved_id):
             raise RuntimeError("Unknown saved id type: %s" % saved_id[0])
 
     f_is_real_file = _is_real_file(f)
+
+    if not hasattr(f, 'seek'):
```

Comment:
This isn't a complete fix because if `f` is unseekable, one of two things could happen:
1) it doesn't have a seek attribute (what you're testing here)
2) calling seek() can throw a unsupported operation: https://docs.python.org/3/library/io.html#io.UnsupportedOperation

I think it might be good to do a try/except block with f.tell() and f.seek() here:
```
try:
   pos = f.tell()
   f.seek(pos)  # should be a no-op
except Unsupported...:
   // seek isn't supported.
```